apps/src/code-studio/pd/workshop_dashboard/components/survey_results/free_response_section.story.jsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import FreeResponseSection from './free_response_section';
+
+export default storybook => {
+  storybook
+    .storiesOf('FreeResponseSection', module)
```

Comment:
If this is using React-Bootstrap, or styles in `pd.scss`, we should add the [ReactBootStoryBookDecorator](https://github.com/code-dot-org/code-dot-org/blob/58a81962e7f489f56387d755b988b6fb2f007c87/apps/src/code-studio/pd/reactBootstrapStoryDecorator.jsx#L1-L0). See [example](https://github.com/code-dot-org/code-dot-org/blob/d78ceb86001b128a778baf2334ee7cb92bee9216/apps/src/code-studio/pd/form_components/QuestionsTable.story.jsx#L8)